### ISSUE TITLE
mockgen command fails when source code of gomock is contained in vendor directory

### ISSUE THREAD

**Issue by triggetry** [golang/mock#143]
I am using `mockgen` to generate mocks for my interfaces. It all works fine, however as soon as i include the sources for `gomock` into the vendor directory using [golang/dep](https://github.com/golang/dep), the `mockgen` command fails with the following error message:

`Loading input failed: gob: name not registered for interface: "github.com/triggetry/goproj/vendor/github.com/golang/mock/mockgen/model.PredeclaredType"`

Any ideas?
Thanks

**Comment by adampointer** [comment]
I have been banging my head against this one for hours. I saw your issue and I realised exactly what the issue is, so thanks!

Maybe it's the same for you, I use dep to get my dependencies at it want to install the v1.0.0 tag of `github.com/golang/mock` while go getting to get the binary will give you the master branch. I think its the two differing versions which is causing the problem. I changed `Gopkg.toml` to install `master` not the tagged release, and it works. 

I hope this helps you as much as it has helped me!!
